core-bundle/src/Resources/contao/library/Contao/Picture.php

```diff
@@ -64,7 +64,7 @@ class Picture
 	/**
 	 * The image size items collection
 	 *
-	 * @var ImageSizeItemModel[]|Collection
+	 * @var array<ImageSizeItemModel|object>|Collection
```

Comment:
Where does `object` come from? Unit tests only or productional code?

Reply:
The class can operate on objects an the unit tests deliberately test it. But I'm also fine to rewrite the unit tests to use the models instead if we're sure this is no supported real world use case. 

/cc @ausi

Reply:
I think objects are a supported use case. For hardcoded image size items you don’t need to create a model object (which was not easily possible in previous Contao versions AFAIK)